build/clone.sh

```diff
@@ -1,9 +1,7 @@
 #!/usr/bin/env nix-shell
-#!nix-shell -i bash ../shell.nix
+#!nix-shell -i bash ../terraform/shell.nix
```

Comment:
I think this needs a minor fixup?

```suggestion
#!nix-shell -i bash ../shell.nix
```

Reply:
There were a few other instances (e.g. `ci/terraform-apply.sh`) that I fixed up as well and amended.